engine/render/src/render/font_renderer_default.h

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020-2024 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#ifndef DM_FONT_RENDERER_DEFAULT_H
+#define DM_FONT_RENDERER_DEFAULT_H
+
+#include <dlib/math.h>                      // for dmMath::Max
+#include <dlib/utf8.h>                      // for dmUtf8
+
+#include "font_renderer.h"
+#include "font_renderer_private.h"          // for RenderLayerMask
+
+namespace dmRender
+{
+    // Private!
+    // Only here for unit tests
```

Comment:
Maybe put these at the bottom so it's easier to see where the test code ends? And also it's the first thing you see in the file, so  you might think it's more relevant than it is 🤔

Reply:
Not sure what you mean. They're only exposed in the header because the unit test currently uses them.
There are no "public" functions in that file currently, so it's already "last" in the file.